cpp/examples/deck.gl/texture-render.cc

```diff
@@ -76,7 +76,7 @@ auto createDeck(const char* argv[], const wgpu::Device& device, const lumagl::Si
   deckProps->width = size.width;
   deckProps->height = size.height;
 
-  deckProps->renderingOptions = Deck::RenderingOptions{device, device.CreateQueue(), false};
+  deckProps->drawingOptions = new lumagl::AnimationLoop::Options{device, device.CreateQueue()};
```

Comment:
Nit: Not a great name for this prop. We can handle this in API audit. Let's make an issue for API audit and add a bullet for this.

Reply:
Agreed, adding this to the list